scripts/publish.sh

```diff
@@ -71,18 +71,21 @@ for crate in "${TOPOLOGICAL_ORDER[@]}"; do
   done
 done
 
-[ "$1" = --no-dry-run ] || d "Run with --no-dry-run from the merged PR to actually publish"
+[ "$1" = --dry-run ] && d "Nothing more to do with --dry-run"
 
-i "Remove all -git suffixes"
+i "Remove all -git suffixes (if any)"
 sed -i 's/-git//' $(git ls-files '*/'{Cargo.{toml,lock},CHANGELOG.md})
 if [ -n "$(git status -s)" ]; then
   i "Commit release"
   git commit -aqm'Release all crates'
   t "Create a PR from this release commit"
-  d "Then re-run from the merged PR"
+  d "Then re-run from the merged PR with --no-dry-run"
```

Comment:
For more precaution, let's actually not suggest this and let the maintainer get the error below line 88 telling to run with `--no-dry-run`.